src/components/PageWithSideMenu.tsx

```diff
@@ -71,9 +70,14 @@ interface LinkButtonProps extends ButtonProps, Partial<Pick<LinkProps, 'to'>> {
   isSelected?: boolean;
 }
 
-export const LinkButton = ({ isSelected, sx, ...rest }: LinkButtonProps) => (
+export const LinkButton = ({ isSelected, startIcon, sx, ...rest }: LinkButtonProps) => (
```

Comment:
noen grunn til at dette er lagt til?
```suggestion
export const LinkButton = ({ isSelected, sx, ...rest }: LinkButtonProps) => (
```